### PR TITLE
docs upgrade

### PR DESCRIPTION
**Pull request by esc**
the documentation had started to rot a little, this is a refresh. @sklam can you please create the numba-scipy category on the Discourse and we can then add that link, get this reviewed and merged. Thanks!

### CODE REVIEW

**Comment by sklam** [comment]
Discourse category for numba-scipy: https://numba.discourse.group/c/numba/numba-scipy/15

**Comment by esc** [comment]
Category added in 2b7894d -- please re-review.

**Comment by esc** [comment]
hmmm, no read the docs integration?

File: docs/source/developer/contributing.rst [hunk 1]
```diff
@@ -10,13 +10,13 @@ improvements are encouraged.  If you have questions, don't hesitate to ask them
...
-''''''''''''
+Discourse
+'''''''''
 
-There is a public mailing-list that you can e-mail at numba-users@anaconda.com.
-If you have any questions about contributing to numba-scipy, it is ok to ask
-them on this mailing-list.  You can subscribe and read the archives on
-`Google Groups <https://groups.google.com/a/continuum.io/forum/#!forum/numba-users>`_.
+The Numba community uses Discourse for asking questions and having discussions
+about numba-scip. There are various categories available and it can be reached
```
Comment (sklam) [inline docs/source/developer/contributing.rst:17]:
```suggestion
about numba-scipy. There are various categories available and it can be reached
```

Comment (esc) [reply docs/source/developer/contributing.rst]: Well spotted, thank you!

File: docs/source/developer/contributing.rst [hunk 2]
```diff
@@ -10,13 +10,13 @@ improvements are encouraged.  If you have questions, don't hesitate to ask them
...
+'''''''''
 
-There is a public mailing-list that you can e-mail at numba-users@anaconda.com.
-If you have any questions about contributing to numba-scipy, it is ok to ask
-them on this mailing-list.  You can subscribe and read the archives on
-`Google Groups <https://groups.google.com/a/continuum.io/forum/#!forum/numba-users>`_.
+The Numba community uses Discourse for asking questions and having discussions
+about numba-scipy. There are various categories available and it can be reached
+at: `numba.discourse.group <https://numba.discourse.group/>`_. There is also a
+category for `numba-scipy <https://numba.discourse.group/c/numba/numba-scipy/15>`_.
```
Comment (stuartarchibald) [inline docs/source/developer/contributing.rst:19]:
```suggestion
category for `numba-scipy <https://numba.discourse.group/c/numba/numba-scipy>`_.
```

File: docs/source/developer/contributing.rst [hunk 3]
```diff
@@ -10,13 +10,13 @@ improvements are encouraged.  If you have questions, don't hesitate to ask them
...
+Discourse
+'''''''''
 
-There is a public mailing-list that you can e-mail at numba-users@anaconda.com.
-If you have any questions about contributing to numba-scipy, it is ok to ask
-them on this mailing-list.  You can subscribe and read the archives on
-`Google Groups <https://groups.google.com/a/continuum.io/forum/#!forum/numba-users>`_.
+The Numba community uses Discourse for asking questions and having discussions
+about numba-scipy. There are various categories available and it can be reached
+at: `numba.discourse.group <https://numba.discourse.group/>`_. There is also a
```
Comment (stuartarchibald) [inline docs/source/developer/contributing.rst:18]:
```suggestion
at: `numba.discourse.group <https://numba.discourse.group/>`_. There is a
```

File: docs/source/developer/contributing.rst [hunk 4]
```diff
@@ -10,13 +10,13 @@ improvements are encouraged.  If you have questions, don't hesitate to ask them
 Communication
 -------------
 
-Mailing-list
-''''''''''''
+Discourse
+'''''''''
```
Comment (stuartarchibald) [inline docs/source/developer/contributing.rst:14]:
```suggestion
Contact
'''''''
```
cf. https://github.com/numba/numba/pull/7447/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR79-R80

**Review by stuartarchibald — COMMENTED** [review]
Few minor things to resolve else looks good, thanks for the patch.

**Comment by esc** [comment]
I have made all the requested changes.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch and fixes.

**Pull request merged**